.github/workflows/integration_tests.yml

```diff
@@ -135,6 +135,21 @@ jobs:
         run: |
           choco install openssl -r
 
+      - name: Install OpenSSL (MacOS)
+        if: matrix.target_platform == 'Desktop' &&
+            matrix.ssl_variant == 'openssl' &&
+            startsWith(matrix.os, 'macos')
+        run: |
+          brew install openssl
+
+      - name: Install OpenSSL (MacOS)
```

Comment:
replace MacOS with Linux

Reply:
Do we need this step?  Aren't our linux builds succesfull without it?

Reply:
Good catch. Replaced MacOS with Linux.
I left the linux step for consistency because I dont know how is it finding openssl otherwise. The github runners (ubuntu 18) does not have openssl pre-installed.